public-html/aboutus.php

```diff
@@ -2,15 +2,35 @@
 require $_SERVER['DOCUMENT_ROOT'] . '/../vendor/autoload.php';
 use src\SessionCookie;
 $session = new SessionCookie;
-$session->includeAccess();
```

Comment:
you sure na that removing this will cause no issue

Reply:
Ya include access was just including menus, which is now done by creating object of `Menu` class hence this is not required

Reply:
> Ya include access was just including menus, which is now done by creating object of `Menu` class hence this is not required

Cool, then thats fine